### PR TITLE
fix: flash of grey while hard reloading app detail page

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

- [x] fixes back button style in app detail view
- [x] fixes flash of grey while hard reloading app detail page - ex: https://app.cal.com/apps/hubspot
- [x] fixes untranslated i18n placeholder flash 

button after:
Screenshot 2023-01-18 22:51:37

 

**Environment**: Staging(main branch) / Production

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by zomars — CHANGES_REQUESTED** [review]

File: apps/web/components/apps/App.tsx [hunk 1]
```diff
@@ -314,7 +314,7 @@ export default function App(props: {
   const { t } = useLocale();
 
   return (
-    <Shell large isPublic heading={t("app_store")} backPath="/apps" withoutSeo>
+    <Shell large heading={t("app_store")} backPath="/apps" withoutSeo>
```
[the remark this answers is not in the document]
Comment (zomars) [reply apps/web/components/apps/App.tsx:317]: Try this link in a private window: https://cal-git-fix-appstore-flash-cal.vercel.app/apps/hubspot

[the remark this answers is not in the document]
Comment (G3root) [reply apps/web/components/apps/App.tsx]: the issue was with the kbar. brought back the isPublic prop and pushed some more changes.

**Comment by PeerRich** [comment]
@emrysal can you review this? you have the most experience with the public app store page

**Comment by emrysal** [comment]
One thing that I think we should fix too is the display of the sidebar during load on https://cal-git-fix-appstore-flash-cal.vercel.app/apps/basic - @PeerRich This PR or follow up? (I'd say this PR as it fixes similar issues)

**Comment by G3root** [comment]
@emrysal i think this has been fixed from some other PR. pushed the latest main to the PR and it seem good now https://cal-5whv607bg-cal.vercel.app/apps/basic

**Comment by PeerRich** [comment]
so I guess we can close this?

**Comment by zomars** [comment]
Closing for now

**Pull request closed without merging**